build/LegacyPackages/14.0/Microsoft.Build.Conversion.Core/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
```

Comment:
The license URL should probably point to our license in GitHub.  Right now this link takes you to the .NET Library license: https://www.microsoft.com/net/dotnet_library_license.htm

Should I have it go to https://raw.githubusercontent.com/Microsoft/msbuild/master/LICENSE ?  Maybe even get a fwlink?